float64 x
float64 y
float64 gnss_lat
float64 gnss_lon
---
float64 lat
float64 lon
